chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -70,15 +71,11 @@ namespace {
 
 AutocompleteMatch GetAutocompleteMatchForText(Profile* profile,
                                               const std::u16string& text) {
+  auto* classifier = AutocompleteClassifierFactory::GetForProfile(profile);
```

Comment:
Nit: Maybe just add a comment saying that `AutocompleteClassifierFactory` uses `BraveAutocompleteSchemeClassifier`.

Reply:
Done.